### ISSUE TITLE
pip install r2pipe error on Windows 7

### ISSUE THREAD

**Issue by andreafioraldi** [radare/radare2-r2pipe#42]
When i try to install r2pipe in Windows 7 i get the following error:
```
> python.exe -m pip install r2pipe                                                                                           
Collecting r2pipe                                                                                                            
  Using cached r2pipe-0.9.5.tar.gz                                                                                           
    Complete output from command python setup.py egg_info:                                                                   
    Traceback (most recent call last):                                                                                       
      File "<string>", line 1, in <module>                                                                                   
      File "c:\users\andrea\appdata\local\temp\pip-build-pogrdt\r2pipe\setup.py", line 2, in <module>                        
        import r2pipe                                                                                                        
      File "r2pipe\__init__.py", line 32, in <module>                                                                        
        import fcntl                                                                                                         
    ImportError: No module named fcntl                                                                                       
                                                                                                                             
    ----------------------------------------                                                                                 
Command "python setup.py egg_info" failed with error code 1 in c:\users\andrea\appdata\local\temp\pip-build-pogrdt\r2pipe\   
```   
note: I'm using python 2.7.14 amd64

**Comment by radare** [comment]
Should be fixed in 0.9.6 pls update r2pipe and confirm

**Comment by radare** [comment]
closing for now
